less/header.less

```diff
@@ -0,0 +1,102 @@
+.navigation {
+  max-width: 100%;
+  position: fixed;
+  width: 100%;
+  z-index: 20;
+
+  h3 {
+    font-size: 2.4rem;
+    color: @bg-color;
+  }
+
+  .main-navigation {
+    background-color: @nav-bg;
+    .center;
+    justify-content: space-between;
+    padding: 13px 8%;  
+  }
+
+  .overlay {
+    width: 100%;
+    height: 100%;
+    background-color: @nav-ex-bg;
+
+    .overlay-content {
+      height: 100%;
+      .center;
+      justify-content: space-between;
+      align-items: flex-start;
+      padding: 13px 8%;
+
+      .links {
+        align-self: center;
+
+        a {
+          display: block;
+          font-weight: bold;
+          font-size: 4.8rem;
+          line-height: 85px;
+          color: @bg-color;
+          text-decoration: none;
+
+          &:hover {
+            color: @font-color;
+          }
+        }
+      }
+
+    }
+  }
+
+  .hidden {
+    display: none;
+  }
+}
+
+.extend {
+  height: 100%;
+}
+
+.main-header {
+  background-image: url(../img/home/home-jumbotron.png);
+  background-repeat: no-repeat;
+  background-position: center bottom;
+  background-size: 100% 100%;
+  height: 650px;
+
+  @media @phone {
+    background-image: url(../img/home/home-mobile-jumbotron.png);
+    height: auto;
+    background-size: 100%;
+  }
+
+  .title {
+    padding-top: 327px;
+
+    @media @phone {
+      padding: 80px 0 47px 0%;
+      margin-left: 6%;
```

Comment:
I'm getting a horizontal scroll bar on mobile, and this margin-left appears to be the culprit. Did you mean to put padding here instead?